crates/re_space_view_spatial/src/heuristics.rs

```diff
@@ -186,7 +186,7 @@ fn update_depth_cloud_property_heuristics(
                 properties.backproject_radius_scale = EditableAutoValue::Auto(1.0);
             }
 
-            entity_properties.update(ent_path.clone(), properties);
+            entity_properties.overwrite_properties(ent_path.clone(), properties);
```

Comment:
This seems like the opposite of what we want.  If an entity has both a depth heuristic and a pinhole heuristic, only one of these is going to end up being applied.

Reply:
I don't think that's true. Each time we first retrieve `properties` via `entity_properties.get` which will have the updated value. None of the things we set here overwrite each other directly

Reply:
Ahh -- I needed to expand to see the context that this is a read-edit-write loop and not just a blind update.  Still feels fragile, but as I've mentioned, hopefully this is all going away soon enough when we split properties into their own components.